DFE-Digital/ecf-engage-and-learn
# Update ECT dashboard text and style

## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDEL-529

## Code review

### Is there anything weird that code reviewer should know?
Nope, everything is straightforward.

## Product review

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Sign in as an ECT and check out the dashboard page
